extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1740,6 +1740,10 @@ export class AmpStoryPage extends AMP.BaseElement {
 
     if (!this.openAttachmentEl_) {
       this.openAttachmentEl_ = buildOpenAttachmentElement(this.element);
+      const attachmentHref = attachmentEl.getAttribute('href');
+      if (attachmentHref) {
+        this.openAttachmentEl_.setAttribute('href', attachmentHref);
+      }
```

Comment:
A comment could be useful here. I could make the nested `if` easier to decipher.